tools/tests/com.liferay.ide.project.core.tests/src/com/liferay/ide/project/core/tests/NewLiferayPluginProjectOp623Tests.java

```diff
@@ -0,0 +1,169 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+
+package com.liferay.ide.project.core.tests;
+
+import com.liferay.ide.project.core.ProjectCore;
+
+import org.eclipse.core.runtime.IPath;
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * @author Terry Jia
+ */
+public class NewLiferayPluginProjectOp623Tests extends NewLiferayPluginProjectOpBase
```

Comment:
Need to override the testNewSDKProjects() to avoid calling the "test with spaces" project.